docs/how-tos/export-data.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "How to export data"
+linkTitle: "Export Data"
+description: "Download data from the Viam app to your local computer using the data client API or the Viam CLI."
+weight: 45
+type: "docs"
+tags: ["data management", "cloud", "sync"]
+icon: true
+images: ["/services/icons/data-capture.svg"]
+aliases:
+  - /manage/data/export/
+  - /data/export/
+  - /services/data/export/
+viamresources: ["sensor", "data_manager"]
+platformarea: ["data", "cli"]
+level: "Beginner"
+date: "2024-09-13"
+# updated: ""  # When the tutorial was last entirely checked
+cost: "0"
+---
+
+You can download machine data from cloud storage to your computer with the Viam CLI or data client API.
+
+{{% alert title="In this page" color="tip" %}}
+
+- [Export data with the Viam CLI](#export-data-with-the-viam-cli)
+- [Manage data with the data client API](#manage-data-with-the-data-client-api)
+
+{{% /alert %}}
+
+## Prerequisites
+
+{{< expand "Install the Viam CLI and authenticate." >}}
+[Install the Viam CLI](/cli/#install), then [authenticate your session with Viam](/cli/#authenticate).
```

Comment:
Please follow the pattern we use in other guides. We want to provide the information here so they don't have to switch pages.